package.json

```diff
@@ -39,9 +38,9 @@
     "winston": "^3.2.1"
   },
   "devDependencies": {
-    "eslint": "^7.11.0",
     "chai": "^4.2.0",
     "chai-http": "^4.3.0",
+    "eslint": "^7.11.0",
```

Comment:
yay for alphabestisation, I haven't been doing it as I'm usually not otherwise touching the things I want to do it to 🤣

Reply:
I know! It's so tempting sometimes, isn't it?